game/domain/builders/sectors.lua

```diff
@@ -31,6 +31,18 @@ function BUILDER.build(idgenerator, player_data)
   local pbody = BODY_BUILDER.buildState(idgenerator, species, 16, 12)
   local pactor = ACTOR_BUILDER.buildState(idgenerator, background, pbody)
 
+  -- generate npcs
+
+  local npcs = {
+    BODY_BUILDER.buildState(idgenerator, "corgi", 11, 11),
+    BODY_BUILDER.buildState(idgenerator, "slime", 9, 13),
+  }
+  npcs[1].dialogue = [[
+  [speed=fast][size=big][style=shake]HELLO[pause=1.5][endl=1][style=none][speed=regular][size=regular]How are you today, [color=red]my friend?[color=regular][pause=1.5][endl=1][speed=slow]...[speed=regular]well I'm feeling totally [style=wave][color=blue][speed=medium]groooooovy[style=none]!
+  ]]
+  npcs[2].dialogue = "howdy, i like pears.[pause=1][endl=1][size=small][speed=fast]dont tell anyone my  [opacity=semi][color=red][speed=medium][style=shake]dark secret"
+
+
```

Comment:
FFS please put these strings somehwere else

Reply:
Its just test text to showcase what the box can do. I'll change for a smaller text, but I won't put it somewhere else until I start working on the actual initial map